### PR TITLE
feature: support multiple ports like stream listen in http and https …

### LINKED ISSUE #1195: feature: support multiple ports like stream listen in http and https listen port.

### PR DESCRIPTION
**Pull request by tzssangglass**
…listen port. (#1195)

fix #1195

### What this PR does / why we need it:

The Apache APISIX listens to a single port by default in http and https, this PR support multiple ports listen.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by membphis** [comment]
CI output:

```
+ ./bin/apisix init
lua: ./bin/apisix:887: attempt to call field 'new' (a nil value)
stack traceback:
	./bin/apisix:887: in function '?'
	./bin/apisix:1154: in main chunk
	[C]: ?
Error: Process completed with exit code 1.

**Review by membphis — APPROVED** [review]

**Comment by tzssangglass** [comment]
someone else can approved?

**Comment by tzssangglass** [comment]
someone else can approved?

File: FAQ_CN.md [hunk 1]
```diff
@@ -220,3 +220,23 @@ curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034
 ```shell
 apisix reload
 ```
+
+## 如何支持http和https多端口监听
+apisix默认监听单端口，如果需要支持http或https监听多端口，修改步骤：
+
+1、修改conf/config.yaml中的http端口监听的配置参数`node_listen`，或https端口监听的参数`ssl.listen_port`，格式如下
+```
```
[the remark this answers is not in the document]
Comment (membphis) [reply FAQ_CN.md:228]: 修改conf/config.yaml中的http端口监
->
修改`conf/config.yaml`中的 HTTP 端口监

File: FAQ_CN.md [hunk 2]
```diff
@@ -220,3 +220,23 @@ curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034
...
+    - 9081
+    - 9082
+  ssl:
+    listen_port:
+      - 9443
+      - 9444
+      - 9445
+```
+
+2、重启APISIX
```
Comment (membphis) [inline FAQ_CN.md:241]: need a space between Chinese and English

File: FAQ.md [hunk 3]
```diff
@@ -270,3 +270,21 @@ If your APISIX node does not open the Admin API, then you can manually load the
 ```shell
 apisix reload
 ```
+
+## How to support multiple ports in http and https listen port？
+
+The Apache APISIX listens to a single port by default. If you need to support http or https to listen to multiple ports, modify the steps:
```
[the remark this answers is not in the document]
Comment (tzssangglass) [reply FAQ.md:276]: ^_^ haha, ok, I use this site for translations, thanks for the guidance!

**Comment by nic-chen** [comment]
need test case for change.

here is an example:

https://github.com/apache/apisix/blob/master/.travis/apisix_cli_test.sh#L156-L170

File: FAQ.md [hunk 4]
```diff
@@ -271,20 +271,29 @@ If your APISIX node does not open the Admin API, then you can manually load the
...
+    
+   ```
     apisix:
       node_listen:
         - 9080
         - 9081
         - 9082
+    ```
+  
+   Handling https requests is similar, Modify the parameter of HTTPS port listen `ssl.listen_port` in `conf/config.yaml`, style like
```
Comment (membphis) [inline FAQ.md:288]: `http` -> `HTTP`
`https` -> `HTTPS`

Comment (membphis) [reply FAQ.md:288]: `Modify` -> `modify`

Comment (membphis) [reply FAQ.md:288]: `style like` -> `for example:`

**Comment by membphis** [comment]
please take a look at the output of CI: https://github.com/apache/apisix/pull/2409/checks?check_run_id=1260519517#step:6:125

```
Scan detected 6 error(s) in 2 file(s):
Reading configuration file [.travis/ASF-Release.cfg]...
Scanning files starting at [./]...
Scan detected 6 error(s) in 2 file(s):
  [./FAQ.md]:
     279: line has trailing whitespace.
     287: line has trailing whitespace.
     289: line has trailing whitespace.
     298: line has trailing whitespace.
  [./FAQ_CN.md]:
     237: line has trailing whitespace.
     239: line has trailing whitespace.

**Comment by tzssangglass** [comment]
> need test case for change.
> 
> here is an example:
> 
> https://github.com/apache/apisix/blob/master/.travis/apisix_cli_test.sh#L156-L170

ok, thx

**Comment by tzssangglass** [comment]
The last commit was on my company's virtual machine. Maybe it caused the security check to fail. In the evening I will submit from my own computer.

**Review by membphis — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**